The-Product-Hub/iReporter
# ft-profile-page-162162150

####What does this PR do?
This PR let's a user who has signed up view their profile and dashboard.
It also lets the user make Intervention and red-flags reports.
Users can also change settings, delete reports, and view their reports history.

####Description of Task to be completed?
Create a profile page that let's user:
-View their profile
-Change settings as desired
-See their records history
-Option to delete records

####How should this be manually tested?
User should be able to view profile when profile or sign in button is clicked

####What are the relevant pivotal tracker stories?
#162162150
